### PR TITLE
create animals table

### PR DESCRIPTION
**Pull request by joskalenda**
## Vet clinic database: create animals table

- [ ] Learning  objectives

    - Create tables in SQL.
    - Insert and query data in SQL.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Agumon', '2020-02-03', 10.23, true, 0);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Gabumon', '2018-11-15', 8.0, true, 2);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Pikachu', '2021-01-7', 15.05, false, 1);
```
Comment (mmsesay) [inline data.sql:6]: - [ ] I found a typo here. Animal `Pikachu` weight should be `15.04`

File: data.sql [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Devimon', '2017-05-12', 11.05, false, 5);
```
Comment (mmsesay) [inline data.sql:7]: - [ ] I found a typo here. Animal Devimon's weight should be 11kg instead of 11.05kg

File: schema.sql [hunk 3]
```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
+
 CREATE TABLE animals (
-    name varchar(100)
-);
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR (50),
```
Comment (mmsesay) [inline schema.sql:6]: - [OPTIONAL] It will be nice if you could increase the value to 255.

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @joskalenda, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by joskalenda** [comment]
Thanks, @mmsesay , I will fix them up right soo

**Pull request merged**